docs/docs/client-data-fetching.md

```diff
@@ -186,11 +186,13 @@ class ClientFetchingExample extends Component {
             <p>Oh noes, error fetching pupper :(</p>
           )}
         </div>
-       </div> {/* highlight-end */}
+        {/* highlight-end */}
+       </div>
     )
   }
 
-  // This data is fetched at run time on the client. // highlight-start
+  // highlight-start
+  // This data is fetched at run time on the client.
```

Comment:
This part seems to work correctly now? But for readability this is great change!